### ISSUE TITLE
.7 SSL issues

### ISSUE THREAD

**Issue by ZachRoberts25** [travetto/travetto#39]
Application starts up with no errors.  If I try to curl the app I get

`curl: (35) error:1400410B:SSL routines:CONNECT_CR_SRVR_HELLO:wrong version number`

in the browser

```This site can’t provide a secure connection localhost uses an unsupported protocol.
ERR_SSL_VERSION_OR_CIPHER_MISMATCH```

any ideas?  

Think this may be something locally I have installed, version mismatch or something

**Comment by ZachRoberts25** [comment]
Verified if I downgrade the application the problem goes away.  Looks like its specific to the 0.7.x versions

**Comment by arciisine** [comment]
The SSL version should be tied to a dependency that was most likely, transitively upgraded.  I've been working with SSL support for a while in 0.7 without issue.  What browser (and version) are you testing with?

**Comment by ZachRoberts25** [comment]
chrome `Version 75.0.3770.100 (Official Build) (64-bit)`
curl `curl 7.54.0 (x86_64-apple-darwin18.0) libcurl/7.54.0 LibreSSL/2.6.5 zlib/1.2.11 nghttp2/1.24.1`
openssl `LibreSSL 2.6.5`

**Comment by arciisine** [comment]
Alright, this was an artifact of me switching from a mac to linux.  Should be fixed in the 0.7.1 release.  let me know if it isn't

**Comment by ZachRoberts25** [comment]
Closing as resolved
